test/extended/authentication/keycloak_client.go

```diff
@@ -133,6 +133,8 @@ func (kc *keycloakClient) CreateUser(username, password string, groups ...string
 type authenticationResponse struct {
 	AccessToken string `json:"access_token"`
 	IDToken     string `json:"id_token"`
+	Error       string `json:"error,omitempty"`
+	ErrorDescription string `json:"error_description,omitempty"`
```

Comment:
This will allow us to surface errors from Keycloak as we encounter them during authentication